ts/Core/Series/DataLabel.ts

```diff
@@ -763,6 +763,13 @@ namespace DataLabel {
                                 dataLabel.css(style).shadow(
                                     labelOptions.shadow
                                 );
+                            } else if (style.width) { // #20499
```

Comment:
Redundant, the issue number is mentioned below
```suggestion
                            } else if (style.width) {
```